### DISCUSSION TITLE
Smoke Produced - Mass loss rate

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#12766] [Q&A]
I have a weird phenomenon that I would need explanation. I am simulating a steady state fire in an apartment room with an hrr that ramps up to 1000 kw in a second and there is a constant size opening in the room. The apartment door opens at 120s and the quantity of smoke that enters the corridor travels a certain distance. However at a certain time in the simulation the quantity of smoke reduces Even though there is no change in the controls. My only translation to this is the weird drop in the mass loss rate products that occurs in the same time of the smoke reduced however I can't explain why this happens.

[HRR.csv](https://github.com/firemodels/fds/files/14917434/HRR.csv)

**Comment by rmcdermo** [comment]
Is your fire going under-ventilated?  Perhaps you have consumed the oxygen in the compartment.

**Reply by ShehabGamrah** [reply]
can the oxygen be consumed if the hrr is constant at 1000kw. And the openings are not closed. Moreover the same opening size is applied to similar models

**Reply by rmcdermo** [reply]
Apparently the hrr is not a constant.  Probably the mass loss is specified and is constant.  But whether that gets converted to hrr depends on the availability of oxygen.  The compartment can become under-ventilated even if there are openings.  You have to take care to ensure you have resolved the openings or otherwise accounted for ventilation in the real scenario.

**Reply by ShehabGamrah** [reply]
Thanks for your support. I mean the hrr fluctuates around the specified value of 1000kw, but doesn’t drastically drop. How can you determine the size of opening for to become under ventilated per scenario. As I have the same size openings for other scenario and the smoke doesn’t decrease as such.

**Reply by rmcdermo** [reply]
Depends on the case, but the McCaffrey, Quintiere, and Harkleroad (MQH) correlation would be a good place to start.
